### ISSUE TITLE
Missing theme token for the border color of a focused project panel item

### ISSUE THREAD

**Issue by joebno** [zed-industries/zed#12723]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

Latest release has added a blue border to focused file/folder in the project panel, with no clear way to remove it.

I've worked through all settings as detailed in https://[zed.dev/schema/themes/v0.1.0.json](https://zed.dev/schema/themes/v0.1.0.json), and nothing changes it.  It was only on preview (can't remember which version it showed up on), and it has just showed up on the latest stable release (Zed 0.138.4).

### Environment

Zed: v0.138.4 (Zed)
OS: macOS 14.5.0
Memory: 16 GiB
Architecture: aarch64

### If applicable, add mockups / screenshots to help explain present your vision of the feature

<img width="297" alt="Screenshot 2024-06-06 at 09 12 19" src="https://github.com/zed-industries/zed/assets/624217/3ea170f9-a2ba-4f33-8a1a-f268bf50cb38">

**Comment by dimitrisnl** [comment]
Based on the discussion here https://github.com/zed-industries/zed/discussions/12725. You can override it by changing `text.accent`.

I believe this is a bug, though, as I would expect a `border` related style.

**Comment by joebno** [comment]
Thanks @dimitrisnl, changing `text.accent` removed it. 

I would second this as a bug, as it should be a border related style - never would have thought to try the text accent to remove a border.

**Comment by danilo-leal** [comment]
Hey @joebno! Clarifying question: what's the problem here? Is it just a visual preference/request, or is there something to the border that's getting in the way of functionality?

**Comment by failable** [comment]
@danilo-leal  I guess the issue is https://github.com/zed-industries/zed/discussions/12725#discussioncomment-9695080 . The 'issue' should be solved if we could have individual setting for this box.

**Comment by danilo-leal** [comment]
@failable ah, thank you! I changed the issue title so its clearer.
